pos_product_sync/models/product_product.py

```diff
@@ -43,6 +47,10 @@ def send_field_updates(self, product_ids, action=''):
         data = {'message': 'update_product_fields', 'action': action, 'product_ids': product_ids}
         self.env['pos.config'].send_to_all_poses(channel_name, data)
 
+    @api.model
+    def from_js(self, product):
```

Comment:
What is it?

Reply:
Left from playing around at the beginning